pymtl3/passes/mamba/PassGroups.py

```diff
@@ -0,0 +1,27 @@
+from .HeuristicTopoPass import HeuristicTopoPass
+from .TraceBreakingSchedTickPass import TraceBreakingSchedTickPass
+from .UnrollTickPass import UnrollTickPass
+
+UnrollSim = [
```

Comment:
When we get this working, we need to make this consistent with the main pass group file.

Reply:
I can do it in this PR